micro-ui/web/micro-ui-internals/packages/react-components/src/hoc/InboxSearchComposer.js

```
@@ -327,7 +327,7 @@ const InboxSearchComposer = ({configs,headerLabel,additionalConfig,onFormValueCh
                 {/* One can use this Parent to add additional sub parents to render more sections */}
             </div>
             </div>   
-            {showToast && <Toast label={showToast.label} error={showToast?.isError} isDleteBtn={true} onClose={()=>setShowToast(null)}></Toast>}
+            {showToast && <Toast label={showToast.label} type={showToast?.type} isDleteBtn={true} onClose={()=>setShowToast(null)}></Toast>}
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)